source/shared/cpp/ObjectModel/BaseActionElement.h

```diff
@@ -38,6 +38,8 @@ class BaseActionElement
     Json::Value m_additionalProperties;
 
 protected:
+    void PopulateKnownPropertiesSet();
```

Comment:
>void PopulateKnownPropertiesSet() [](start = 4, length = 33)

Can move this to private

Reply:
(Here and throughout)

---
In reply to: [171981106](https://github.com/Microsoft/AdaptiveCards/pull/1179#discussion_r171981106) [](ancestors = 171981106)